monkey_island/cc/services/config.py

```diff
@@ -504,6 +504,13 @@
                             },
                             "default": [],
                             "description": "List of NTLM hashes to use on exploits using credentials"
+                        },
+                        "exploit_ssh_keys": {
```

Comment:
Roping in @itaymmguardicore

Reply:
Preloading ssh keys is of no use. Should I figure a way to remove it from ui?

Reply:
No, if @itaymmguardicore has an idea it'll be nice to fix but really not important